test/webdriverio/customCommands.js

```diff
@@ -10,10 +10,11 @@ browser.addCommand("login", function(serviceName) {
     .setValue(service.login.usernameField, service.login.username)
     .setValue(service.login.passwordField, service.login.password)
     .click(service.login.submitButton)
+    .pause(1000)
```

Comment:
It's not needed here. We just need to pause and allow time for server to process the request.